apps/e2e/action-bars-storybook-e2e/src/e2e/summary-action-bar.component.cy.ts

```diff
@@ -12,84 +12,96 @@ Cypress.on(
+              );
+
+              cy.skyReady('app-summary-action-bar');
+
+              cy.get('.sky-summary-action-bar')
+                .should('exist')
+                .should('be.visible');
+
+              if (style === 'modal') {
+                cy.get('sky-modal-content')
+                  .should('exist')
+                  .should('be.visible')
+                  .click();
+              }
+
+              cy.skyVisualTest(
+                `summaryactionbarcomponent-summaryactionbar--summary-action-bar-${style}-${width}-${theme}`,
+                {
+                  width: width,
+                  capture: 'viewport',
+                },
+              );
+            });
+
+            // Mobile width only/modal specific tests
+            if (
+              E2eVariations.MOBILE_WIDTHS.includes(width) ||
+              style === 'modal'
+            ) {
+              it(`should render the component at width ${width} and with a collapsed summary (${style})`, () => {
                 cy.viewport(width, height).visit(
                   `/iframe.html?globals=theme:${theme}&id=summaryactionbarcomponent-summaryactionbar--summary-action-bar-${style}`,
                 );
 
                 cy.skyReady('app-summary-action-bar');
 
-                cy.get('.sky-summary-action-bar')
+                cy.get('.sky-summary-action-bar-details-collapse button')
                   .should('exist')
-                  .should('be.visible');
+                  .should('be.visible')
+                  .click();
 
-                if (style === 'modal') {
+                if (style.startsWith('modal')) {
                   cy.get('sky-modal-content')
                     .should('exist')
                     .should('be.visible')
                     .click();
+                } else {
+                  cy.get('app-summary-action-bar')
+                    .should('exist')
+                    .should('be.visible')
+                    .click();
                 }
 
+                cy.get('.sky-summary-action-bar')
+                  .should('exist')
+                  .should('be.visible');
+
                 cy.skyVisualTest(
-                  `summaryactionbarcomponent-summaryactionbar--summary-action-bar-${style}-${width}-${theme}`,
+                  `summaryactionbarcomponent-summaryactionbar--summary-action-bar-${style}-${width}-${theme}-collapsed-summary`,
                   {
                     width: width,
                     capture: 'viewport',
                   },
                 );
               });
-
-              // Mobile width only/modal specific tests
               if (
-                E2eVariations.MOBILE_WIDTHS.includes(width) ||
-                style === 'modal'
+                style !== 'single-error' &&
+                style !== 'overflow-errors' &&
+                style !== 'modal-errors'
```

Comment:
this check is also new. this block just takes a screenshot with the "secondary action" dropdown opened. i wanted to try and minimize screenshots